src/DynamicDataTable.jsx

```diff
@@ -236,20 +236,29 @@ class DynamicDataTable extends Component {
     }
 
     renderHeader(field) {
-        const props = this.props;
+        const { orderByField, orderByDirection, allowOrderingBy, excludeOrderingBy, changeOrder } = this.props;
         let orderByIcon = '';
 
-        if (props.orderByField === field.name) {
-            orderByIcon = (props.orderByDirection === 'asc') ? '↓' : '↑';
+        if (orderByField === field.name) {
+            orderByIcon = (orderByDirection === 'asc') ? '↓' : '↑';
         }
         
-        const canOrderBy = props.allowOrderingBy.length === 0 || props.allowOrderingBy.includes(field.name);
+        const canOrderBy = (
+            (allowOrderingBy.length === 0 || allowOrderingBy.includes(field.name))
+            && !excludeOrderingBy.includes(field.name)
```

Comment:
In the event that a DataTable has `allowOrderingBy={['name']} excludeOrderingBy={['name']}`. This would essentially prioritise the exclusion, as if it's present within that array it will not be displayed at all, is this correct? 

This would still then only show the columns specified within the `allowOrderingBy`, and not fall back to the default behaviour of allowing ordering by every column.

(Not that this should ever _happen_, but just to make sure the logic is :ok_hand:)

Reply:
Having both `allowOrderingBy` and `excludeOrderingBy`, disables ordering by the field over allowing it.

I believe this is how it should be, as allowing could cause 500 errors on the server when using with Laravel whereas disabling just means that it isn't triggered and removing the prop would resolve the issue.

Reply:
Definitely something we can look into. This PR needs to be merge quickly, so we can address any logic changes here in the future if needed.

The documentation will state the following, which should cover this.

> Typically, the `allowOrderingBy` and `disallowdOrderingBy` props should not be used simultaneously, as this could cause unexpected behaviour.